sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -1063,6 +1065,45 @@ def test_tostring_kvs_empty_delimeter(self):
       assert_that(result, equal_to(["one1", "two2"]))
 
 
+class LogElementsTest(unittest.TestCase):
+  @pytest.fixture(scope="function")
+  def _capture_stdout_log(request, capsys):
+    with TestPipeline() as p:
+      result = (p | beam.Create([
+          TimestampedValue(
+              "event",
+              datetime(2022, 10, 1, 0, 0, 0, 0,
+                        tzinfo=pytz.UTC).timestamp()),
+          TimestampedValue(
+              "event",
+              datetime(2022, 10, 2, 0, 0, 0, 0,
+                        tzinfo=pytz.UTC).timestamp()),
+      ]) \
```

Comment:
nit: it's generally discouraged to use line continuation symbol. It should be not necessary here given you have parentheses, which ensure statement continuation.

Reply:
Oops, those were leftovers from testing. Removed.